api/resources_portal/test/views/test_attachment.py

```diff
@@ -36,19 +40,40 @@ def setUp(self):
         self.admin = UserFactory()
         self.admin.is_staff = True
 
+        # Cleanup the attachments test directory so there's no files
+        # from previous tests:
+        for directory_name in os.listdir(settings.LOCAL_FILE_DIRECTORY):
```

Comment:
Do we want to convert existing end-to-end tests to use real files? I imagine we do. If so, it would be great if this could be made into a function in the "mocks.py" file (renamed to utils.py in my update-attachments PR). This would allow it to be reused.